cypress/e2e/cloud/flows.test.ts

```diff
@@ -411,7 +411,7 @@ describe('Flows with queryBuilderUseMetadataCaching flag on', () => {
 
     cy.getByTestID('time-machine-submit-button').should('be.visible')
 
-    const clone = `${flowName} (cloned at 11-30-2018:00:00:00)`
+    const clone = `${flowName} (cloned at 2018-11-30 00:00:00)`
```

Comment:
Why do you need to validate a specific date?

The clone's name is a valid date, not a specific date. Test for a valid date, which can be any date. Not a specific date.